src/titlepane/TitlePane.ts

```diff
@@ -73,6 +79,30 @@ export class TitlePaneBase<P extends TitlePaneProperties = TitlePaneProperties>
 		}
 	}
 
+	protected animate(effects: any[]) {
+		const { open = true } = this.properties;
+		this.meta(WebAnimation).animate('content', {
+			id: this._id,
+			effects,
```

Comment:
you should be able to pass `this.getAnimationKeyframes` here and change it to `private`. No need to call it before hand and pass the `effects[]` in.

Reply:
This was so I would only need to call `this.getAnimationKeyframes` once. It's also used to set initial styles here: https://github.com/dojo/widgets/pull/475/files/a0b4c8fca97cc8b9dd4c778e7b4a0b68f1aa25c9#diff-6c2953e8f663b684ca1cfaf618e9b404R185